fatek/target.py

```diff
@@ -40,8 +44,16 @@ def read_all(self, count):
         number = self.symbol.offset
         current_value = self.current_value
 
-        if target in ['Y', 'X', 'M', 'S'] or (current_value == False and target in ['T', 'C']):
-            return self.client.read_coils(number, count).bits  # (start coil, number of readed bits)
-
-        elif target in ['R', 'D'] or (current_value == True and target in ['T', 'C']):
-            return self.client.read_holding_registers(number, count).registers  # (start, number of readed bits)
+        if (
```

Comment:
no, revert it. If you want to simplify conditions, put they into separate variables. I find it hard to read